src/Schema/Table.php

```diff
@@ -26,6 +26,9 @@ class Table extends AbstractAsset
     /** @var Column[] */
     protected $_columns = [];
 
+    /** @var string[] */
+    protected array $_renamedColumns = [];
```

Comment:
I would give as little access as possible to this. We can always loosen access afterwards if required. Also, let's loose the underscore.

```suggestion
    private array $renamedColumns = [];
```